src/lib/ng-package-format/entry-point.ts

```diff
@@ -172,7 +172,9 @@ export class NgEntryPoint {
    *
    * @link https://github.com/webpack/webpack/tree/master/examples/side-effects
    */
-  public get hasSideEffects(): boolean {
-    return this.packageJson['sideEffects'] === true;
+  public get sideEffects(): boolean {
```

Comment:
The return type here is incorrect. I should be `boolean | sting[]`

Why not simply?
```ts
return this.packageJson['sideEffects'] || false;
```

Reply:
Good catch, fixed 😄